File: plugins/flutter_audio_query/ios/Classes/sortingtypes/PlaylistSortType.swift
public enum PlaylistSortType: Int {

    /**
     * The Default sort order for playlist. DEFAULT value the playlist
     * query will return playlists sorted by alphabetically
     */
    case DEFAULT

    /**
     * The most recent playlists will come first in playlist queries.
     *
     */
    case NEWEST_FIRST
    /**
     * The most old playlists will come first in playlist queries.
     */
    case LDEST_FIRST
}

File: plugins/flutter_audio_query/lib/src/playlist_sort_type.dart
part of flutter_audio_query;

/// Enum that define values used to sort playlists.
enum PlaylistSortType {
  /// The Default sort order for playlist. DEFAULT value the playlist
  /// query will return playlists sorted by alphabetically
  DEFAULT,

  /// The most recent playlist will come first with this
  /// sort type param.
  NEWEST_FIRST,

  /// The most old playlists will come first with this
  /// sort type param.
  OLDEST_FIRST
}
